pvlib/test/test_modelchain.py

```diff
@@ -33,8 +33,8 @@ def cec_dc_snl_ac_system(sam_data):
     module_parameters['b'] = 0.05
     module_parameters['EgRef'] = 1.121
     module_parameters['dEgdT'] = -0.0002677
-    inverters = sam_data['cecinverter']
-    inverter = inverters['ABB__MICRO_0_25_I_OUTD_US_208_208V__CEC_2014_'].copy()
+    inverters = sam_data['adrinverter']
```

Comment:
I'm only now realizing that this change to the ``cec_dc_snl_ac_system`` fixture makes the name no longer correct. I do think it would be worthwhile to make a second test called ``cec_dc_adr_ac_system`` or something like it. Then you can revert the change to the ``test_dc_models`` fixture, while also updating the ``test_ac_models`` ``ac_systems`` dictionary.